### PR TITLE
DDP-7174_user_settings 

### PR DESCRIPTION
**Pull request by pegahtah**
user settings changes to match new user settings

### CODE REVIEW

File: ddp-workspace/projects/ddp-dsm-ui/src/app/onc-history-detail/onc-history-detail.component.html [hunk 1]
```diff
@@ -199,10 +199,7 @@
     <tfoot>
     <tr>
       <td [attr.colspan]="10 + (settings['oD'] == undefined? 0: settings['oD'].length)">
-        <ng-container *ngIf="getRole().getUserSetting().getDateFormat() === 'yyyy-MM-dd'">
-          * unknown dates are allowed. Just enter date as YYYY-MM or YYYY
-        </ng-container>
-        <ng-container *ngIf="getRole().getUserSetting().getDateFormat() === 'MM/dd/yyyy'">
+        <ng-container>
```
Comment (pegahtah) [inline ddp-workspace/projects/ddp-dsm-ui/src/app/onc-history-detail/onc-history-detail.component.html]: we are not having different date formats anymore

File: ddp-workspace/projects/ddp-dsm-ui/src/app/user-setting/user-setting.component.html [hunk 2]
```diff
@@ -34,44 +34,13 @@ <h3 class="Color--primary Line--Break">{{additionalMessage}}</h3>
...
-          <td>Enter date format</td>
-          <td>
-            <mat-select [(ngModel)]="userSetting.dateFormat">
-              <mat-option value="yyyy-MM-dd">yyyy-MM-dd</mat-option>
-              <mat-option value="MM/dd/yyyy">MM/dd/yyyy</mat-option>
-            </mat-select>
-          </td>
-          <td></td>
-          <td></td>
-        </tr>
```
Comment (pegahtah) [inline ddp-workspace/projects/ddp-dsm-ui/src/app/user-setting/user-setting.component.html:74]: They are not configurable anymore

File: ddp-workspace/projects/ddp-dsm-ui/src/app/user-setting/user-setting.model.ts [hunk 3]
```diff
@@ -1,28 +1,21 @@
...
-      this.rowSet0 = rowSet0;
-    }
-    if (rowSet1 !== 0) {
-      this.rowSet1 = rowSet1;
-    }
-    if (rowSet2 !== 0) {
-      this.rowSet2 = rowSet2;
-    }
-    if (this.dateFormat != null) {
-      this.dateFormat = dateFormat;
```
Comment (pegahtah) [inline ddp-workspace/projects/ddp-dsm-ui/src/app/user-setting/user-setting.model.ts:24]: All should be set to default for everyone now.

File: ddp-workspace/projects/ddp-dsm-ui/src/app/onc-history-detail/onc-history-detail.component.html (hunk 1 above)
Comment (alextanlan) [inline ddp-workspace/projects/ddp-dsm-ui/src/app/onc-history-detail/onc-history-detail.component.html]: I suppose there is no need to keep an empty <ng-container> tag.

```
<td ...>
  <ng-container>
      ...
  </ng-container>
</td>
```
=> 
```
<td ...>
  ...
</td>
```

File: ddp-workspace/projects/ddp-dsm-ui/src/app/user-setting/user-setting.model.ts [hunk 4]
```diff
@@ -1,28 +1,21 @@
-import { Utils } from '../utils/utils';
+import {Utils} from '../utils/utils';
 
 export class UserSetting {
```
Comment (alextanlan) [inline ddp-workspace/projects/ddp-dsm-ui/src/app/user-setting/user-setting.model.ts:3]: The class (default/init values) can be shortened, like:
```
import {Utils} from '../utils/utils';

export class UserSetting {
  rowSet0 = 10;
  rowSet1 = 25;
  rowSet2 = 50;
  dateFormat = Utils.DATE_STRING_IN_CVS;

  constructor( public rowsPerPage = 10,
               public defaultTissueFilter: string,
               public defaultParticipantFilter: string
  ) {
    // if value is not 0 fill with actual value from user settings table
    if (rowsPerPage && rowsPerPage !== 0) {
      this.rowsPerPage = rowsPerPage;
    }
  }

  static parse( json ): UserSetting {
    return new UserSetting( json.rowsOnPage, json.defaultTissueFilter, json.defaultParticipantFilter );
  }

  public getRowsPerPage(): number {
    return this.rowsPerPage;
  }

  public getRowSet0(): number {
    return this.rowSet0;
  }

  public getRowSet1(): number {
    return this.rowSet1;
  }

  public getRowSet2(): number {
    return this.rowSet2;
  }
}

```

**Pull request closed without merging**